ic-os/components/misc/guestos-recovery/guestos-recovery-engine/guestos-recovery-engine.sh

```diff
@@ -40,7 +40,7 @@ perform_recovery() {
 
         echo "Attempting to download recovery artifact from $recovery_url"
 
-        if curl -L --fail -o "recovery.tar.zst" "$recovery_url"; then
+        if curl --proto '=https' -L --proto-redir '=https' --tlsv1.2 -sS --fail -o "recovery.tar.zst" "$recovery_url"; then
```

Comment:
In scripts I suggest always using the long form arguments like:
```suggestion
        if curl --proto '=https' -L --proto-redir '=https' --tlsv1.2 --silent --show-error --fail -o "recovery.tar.zst" "$recovery_url"; then
```
Same below.

Reply:
Thanks! https://github.com/dfinity/ic/pull/6784/commits/10f2cfb7987a083ea459fb4deed50dbbb2fb1c10